bin/clean_topics_vs_practices.py

```diff
@@ -0,0 +1,28 @@
+#!python3
+import json
+
+
+def main():
+    with open("config.json") as f:
+        config = json.load(f)
+
+    concepts = [c['slug'] for c in config['concepts']]
+    concepts.sort()
+
+    for practice_exercise in config['exercises']['practice']:
+        if practice_exercise['topics'] is None:
+            continue
+
+        for topic in practice_exercise['topics']:
+            if topic in concepts:
+                practice_exercise['topics'].remove(topic)
+                practice_exercise['practices'].append(topic)
```

Comment:
While this only performs a single iteration over the topics, which is good, it's messy to edit the list of topics in the middle of iterating over it. I'd prefer a pair of list comprehensions, so it's clear that the logic is sound.

```suggestion
        practice_exercise['practices'].extend((topic for topic in practice_exercise['topics'] if topic in concepts))
        practice_exercise['topics'] = [topic for topic in practice_exercise['topics'] if topic not in concepts]
```

Reply:
This was something I was also unsure about. I like the suggestion, thank you.